graduate-common/src/tests/prereq_loader.test.js

```diff
@@ -1,6 +1,6 @@
-const prereq_loader = require("../src/prereq_loader");
+const prereq_loader = require("../prereq_loader");
```

Comment:
what do you think about moving the `tests` outside of `src` to the top-level of the `graduate-common` package?

Reply:
that makes sense to me, will update the PR